src/System.Private.CoreLib/shared/System/String.cs

```diff
@@ -472,6 +474,13 @@ public static bool IsNullOrWhiteSpace(string value)
             return true;
         }
 
+        /// <summary>
+        /// Returns a reference to the first element of the String. If the string is null, an access will throw a NullReferenceException.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Prefer fully-qualifying these here rather than introducing using aliases at the top of the file.